extension/json/json_serializer.cpp

```diff
@@ -216,4 +223,13 @@ void JsonSerializer::WriteDataPtr(const_data_ptr_t ptr, idx_t count) {
 	PushValue(val);
 }
 
```

Comment:
Introduces a function to write a timestamp value to JSON, formatting it according to a specified format before adding it to the JSON document.